.cirrus.yml

```diff
@@ -21,6 +21,9 @@ task:
     JOB_SKIP_CI_SPECS: |
       components/perf-tools/likwid/SPECS/likwid.spec
   test_script: |
+    # Remove OpenPBS, if it is installed as a dependency. The tests use Slurm Resource Manager
+    dnf -y remove openpbs-*-ohpc || true
+    rm -f /etc/pbs.conf || true
```

Comment:
What about doing it only once in `tests/ci/setup_slurm_and_run_tests.sh`?

`rm -f /etc/pbs.conf` does not need `|| true`. It will always return `0`.

Reply:
Done!